bucket/butterflow-ui.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "1.1.0",
+    "description": "Graphical user interface for butterflow, a video manipulation tool.",
+    "homepage": "https://github.com/wagesj45/butterflow-ui",
+    "license": "MIT",
+    "architecture": {
```

Comment:
![image](https://user-images.githubusercontent.com/13260377/67269927-e7ffbc00-f4b7-11e9-8ff4-58f9391739f6.png)

Remove architecture property

Reply:
The butterflow is 64bit...

Reply:
butterflow requires 64-bit.

Reply:
![image](https://user-images.githubusercontent.com/16755704/67270788-0f30aa80-f505-11e9-94b6-d5e6de72a779.png)